webapp/src/components/incoming_calls/hooks.ts

```diff
@@ -236,13 +237,20 @@ export const useNotification = (call: IncomingCallNotification) => {
 
 export const useGetCallerNameAndOthers = (call: IncomingCallNotification, splitAt: number) => {
     const {formatMessage, formatList} = useIntl();
+    const dispatch = useDispatch();
     const teammateNameDisplay = useSelector(getTeammateNameDisplaySetting);
     const caller = useSelector((state: GlobalState) => getUser(state, call.callerID));
     const currentUser = useSelector(getCurrentUser);
     const doGetProfilesInChannel = makeGetProfilesInChannel();
     const gmMembers = useSelector((state: GlobalState) => doGetProfilesInChannel(state, call.channelID));
     const callerName = displayUsername(caller, teammateNameDisplay, false);
 
+    useEffect(() => {
+        if (call.channelID) {
+            dispatch(getProfilesInChannel(call.channelID, 0));
```

Comment:
Not sure why this was needed all of the sudden, likely as a result of some changes on webapp but without it, `gmMembers` would be consistently empty.

Reply:
hmm, interesting.
Nice that the tests caught it though!